src/platform/process.cpp

```diff
@@ -245,5 +249,97 @@ namespace Toolbox::Platform {
 
         return true;
     }
+    std::string GetWindowTitle(LowWindow window) {
+        Display* display = XOpenDisplay(0);
+        XTextProperty title;
+        XGetWMName(display, (Window)window, &title);
+        char** stringList;
+        int numStrings;
+        if (!XTextPropertyToStringList(&title, &stringList, &numStrings)) {
+            return "";
+        }
+        if (numStrings < 1){
+            return "";
+        }
+        std::string stringTitle(stringList[0]);
+        XFreeStringList(stringList);
+        return stringList[0];
+    }
+    bool ForceWindowToFront(LowWindow window) {
+        Display* display = XOpenDisplay(0);
+        return XRaiseWindow(display, (Window)window);
+    }
+    bool ForceWindowToFront(LowWindow window, LowWindow target) {
+        // TODO: Is this the right behavior? Not exactly sure how this
+        // is supposed to work.
+        return ForceWindowToFront(window) && ForceWindowToFront(target);
```

Comment:
The intended behavior is that `window` will be forced in front of _at least_ `target`. Although it is much less ideal, a fallback behavior is to indiscriminately force `window` in front of every open window.

Reply:
Ah okay, so what I have here is wrong, this will force target in front of window. I can't find any X11 documentation on bringing a window in front of another without bringing it to the front, so I'll try the fallback for now.